src/vision/status_quo/barbara_builds_an_async_executor.md

```diff
@@ -0,0 +1,161 @@
+# 😱 Status quo stories: Barbara builds an async executor
+
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+## The story
+
+Barbara wants to set priorities to the tasks spawned to the executor. However, she finds no existing async executor provides such a feature so she decided to build her own async executor.
+
+First, Barbara found [crossbeam-deque](https://crates.io/crates/crossbeam-deque) provides work-stealing deques of good quality. She could use it to build task schedulers. It is not the main part of the story. Each working thread can get a task from the scheduler and poll it in loop.
```

Comment:
This paragraph says she _could_ use it to build a scheduler, and below the existence of the scheduler is assumed. Maybe this paragraph should say she _does_ use it to build a scheduler. Also, it could briefly name the operations of that scheduler so the reader knows what `send` is.

Reply:
```suggestion
First, Barbara found [crossbeam-deque](https://crates.io/crates/crossbeam-deque) provides work-stealing deques of good quality. She decides to use it to build task schedulers. It is not the main part of the story. She plans for each working thread to have a loop which repeatedly gets a task from the deque and polls it.
```

Reply:
Ah yes, we can assume that she has made a task scheduler before building the executor.

Reply:
Nit: I'm not sure the sentence "It is not the main part of the story." is actually helpful. We could probably just remove it.